docs/shared/getting-started/nx-setup.md

```diff
@@ -22,41 +26,49 @@ npx create-nx-workspace --preset=core
 npx create-nx-workspace --preset=ts
 ```
 
-Some presets set up applications, e2e tests, etc.
+Some presets are set up to build apps with common frameworks
 
 ```bash
+npx create-nx-workspace --preset=@nrwl/remix
+
 npx create-nx-workspace --preset=react
+
 npx create-nx-workspace --preset=react-native
+
 npx create-nx-workspace --preset=angular
 ```
 
-## Add Nx to an Existing Project
+## Install Nx CLI
 
-If you have an existing Lerna or Yarn monorepo, you can gain the benefits of Nx's computation cache and distributed task execution without modifying the file structure by running this command:
+To make the developer experience nicer, you may want to install the Nx CLI globally.
 
 ```bash
-npx add-nx-to-monorepo
+npm install -g nx
 ```
 
-If you have an existing Create React App project, you can gain the benefits of Nx's computation cache and distributed task execution without modifying the file structure by running this command:
+Optionally, you can use `npx nx` or `yarn nx` to run the CLI.
 
-```bash
-npx cra-to-nx
-```
+## Add Nx to an Existing Project
 
-For more information on adding Nx to an existing repository see the [migration guide](/migration/migration-cra)
+// (Placeholder) is this only for Lerna or Yarn monorepos?
 
-## Install Nx CLI
+If you have an existing Lerna or Yarn monorepo, you can gain the benefits of Nx's computation cache and distributed task execution without modifying the file structure by running the following command:
```

Comment:
we call out Lerna/Yarn workspaces but does it matter? 
what if someone has an existing repo without anything and wants to convert to nx?
is this command only valid for lerna/yarn, could we point to a guide of other existing repos?